app/src/main/res/layout/categories_list_fragment.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.percent.PercentRelativeLayout
```

Comment:
If the children views don't have a `layout_widthPercent` this can be a regular `RelativeLayout`